#!/usr/bin/env python 

PACKAGE = "dynamic_cfg" 
from dynamic_reconfigure.parameter_generator_catkin import * 
gen =ParameterGenerator() 




gen.add( "sensor_height", double_t, 0, "sensor_height", 2.2 , 1, 3) 
gen.add( "groundScanInd", int_t, 0, "groundScanInd", 15 , 0, 31)
gen.add( "min_delta_height_", double_t, 0, "min_delta_height_", 0.1 , 0, 0.5) 
gen.add( "local_max_slope",double_t, 0, "local_max_slope", 5.0 , 0, 30)
gen.add( "max_delta_radius_", double_t, 0, "max_delta_radius_", 0.2 , 0, 1.0)
gen.add( "global_max_slope", double_t, 0, "global_max_slope", 1.8 ,0,30) 



exit(gen.generate(PACKAGE, "dynamic_cfg", "SegGroundLine")) 